app/src/main/java/com/developersbreach/composeactors/ui/screens/movieDetail/MovieDetailsUI.kt

```diff
@@ -151,17 +220,19 @@ private fun GetBottomSheetContent(
 private fun MovieDetailsUILightPreview() {
     ComposeActorsTheme(darkTheme = false) {
         MovieDetailsContent(
-            navigateUp = {},
-            selectBottomSheetCallback = {},
-            showFab = remember { mutableStateOf(true) },
-            openMovieDetailsBottomSheet = { Job() },
             uiState = MovieDetailsUIState(
                 movieData = fakeMovieDetail,
                 similarMovies = fakeMovieList(),
                 recommendedMovies = fakeMovieList(),
                 movieCast = fakeMovieCastList(),
-                isFetchingDetails = false
-            )
+                isFetchingDetails = false,
+                movieProviders = null
```

Comment:
We need previews. Also apply this change in other places. Setting null to any property might not load the whole preview itself.
```suggestion
                movieProviders = MovieProvider(
                    arrayListOf(Flatrate("", 1, ""))
                )
```